demos/cedarling-js-examples/common/ui/theme.css

```
@@ -0,0 +1,145 @@
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background: var(--bg-app);
+  color: var(--text-primary);
+  font-family: "Lato", "Open Sans", Arial, sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)